extension-impl/remoting-bolt/src/main/java/com/alipay/sofa/rpc/transport/bolt/BoltClientConnectionManager.java

```diff
@@ -18,177 +18,39 @@
 
 import com.alipay.remoting.Connection;
 import com.alipay.remoting.Url;
-import com.alipay.remoting.exception.RemotingException;
 import com.alipay.remoting.rpc.RpcClient;
-import com.alipay.sofa.rpc.base.Destroyable;
-import com.alipay.sofa.rpc.common.annotation.VisibleForTesting;
-import com.alipay.sofa.rpc.common.utils.CommonUtils;
-import com.alipay.sofa.rpc.common.utils.NetUtils;
-import com.alipay.sofa.rpc.context.RpcRuntimeContext;
-import com.alipay.sofa.rpc.core.exception.SofaRpcRuntimeException;
-import com.alipay.sofa.rpc.log.Logger;
-import com.alipay.sofa.rpc.log.LoggerFactory;
 import com.alipay.sofa.rpc.transport.ClientTransportConfig;
 
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
-import java.util.concurrent.atomic.AtomicInteger;
-
 /**
- * @author <a href="mailto:zhanggeng.zg@antfin.com">GengZhang</a>
+ * @author bystander
+ * @version $Id: BoltClientConnectionManager.java, v 0.1 2019年01月29日 11:58 bystander Exp $
  */
-class BoltClientConnectionManager {
-
-    /**
-     * slf4j Logger for this class
-     */
-    private final static Logger                            LOGGER               = LoggerFactory
-                                                                                    .getLogger(BoltClientConnectionManager.class);
+public interface BoltClientConnectionManager {
```

Comment:
it' better to change to an abstract class, and move common logic such as `addHook ` to this class.

Reply:
ok